Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -809,6 +817,41 @@ public class RealmServer: NSObject {
     @objc public func createApp() throws -> AppId {
         try createAppForBSONType("string")
     }
+
+    // Retrieve MongoDB Realm AppId with ClientAppId using the Admin API
+    public func retrieveAppId(clientAppId: String) throws -> String {
+        guard let session = session else {
+            throw URLError(.unknown)
+        }
+
+        let appsListInfo = try session.apps.get().get()
```

Comment:
Why is this `get().get()`?

Reply:
first get() use the `GET` as the https method, and the second one returns the success as a throwing expression instead of using a block here